core/src/main/java/org/geysermc/geyser/network/netty/GeyserServer.java

```diff
@@ -222,6 +224,20 @@ public BedrockPong onQuery(InetSocketAddress inetSocketAddress) {
             pong.subMotd(config.getBedrock().secondaryMotd());
         }
 
+        //Moved to prevent overriding values set in the ping event.
```

Comment:
Reword "Moved" to "Placed here" just because the moved part isn't really important after a while sitting here in the code.

Also do keep a space in your comments, please: `// Placed here...`